### PR TITLE
fix(combox and paginatedtable): add ability to provide link for tooltip

### PR DESCRIPTION
**Pull request by jondauz**
Make icon for tooltip clickable.

### CODE REVIEW

**Review by dbhaasis — APPROVED** [review]
Only question is if clicking on the icon rather than hover is supposed to follow link or not. I think it will follow link. Also wondering about how this works for mobile touchscreens.

**Comment by jondauz** [comment]
@dbhaasis On touch devices when clicking the icon it will open the link in a new page, if a link is provide. The tooltip itself will open in both cases.

File: src/js/components/ComboBox/Combobox.tsx [hunk 1]
```diff
@@ -58,6 +59,23 @@ interface Filter {
 
 type Props = PropTypes.InferProps<typeof propTypes>;
 
+const TooltipWrapper = ({ tooltipLink, children }) => {
+  if (!tooltipLink) {
+    return <>{children}</>;
+  }
+
+  return (
+    <a href={tooltipLink} target="_blank" rel="noreferrer">
```
Comment (wnoonan) [inline src/js/components/ComboBox/Combobox.tsx:68]: was looking up noreferrer, do you think noopener would be beneficial here too? https://www.reliablesoft.net/noreferrer-noopener/#noopener

**Review by wnoonan — COMMENTED** [review]
just had a question in a comment but otherwise lgtm!

**Pull request merged**